pkg/config/datafileprojectconfig/config.go

```diff
@@ -127,6 +127,14 @@ func (c DatafileProjectConfig) GetFeatureList() (featureList []entities.Feature)
 	return featureList
 }
 
+// GetExperimentList returns an array of all the experiments
+func (c DatafileProjectConfig) GetExperimentList() (experimentList []entities.Experiment) {
```

Comment:
Can you add a unit test for this method?

Reply:
yes, I missed that